services/movie.py

```diff
@@ -28,13 +33,14 @@
     genres_ids: list = None,
     actors_ids: list = None,
```

Comment:
The default value for `genres_ids` and `actors_ids` is `None`. If you intend to allow empty lists, consider using `Optional[list]` for type hinting or defaulting to an empty list `[]`.